src/Sylius/Bundle/PaymentBundle/Resources/config/services.xml

```diff
@@ -26,17 +26,17 @@
             <argument>Payment methods resolver</argument>
         </service>
 
-        <service id="sylius.composite_payment_methods_resolver" class="Sylius\Component\Payment\Resolver\CompositeMethodsResolver">
+        <service id="sylius.payment_methods_resolver.composite" class="Sylius\Component\Payment\Resolver\CompositeMethodsResolver">
```

Comment:
`sylius.payment_methods_resolver.composite` -> `sylius.payment_methods_resolver` - seems like it is the methods resolver that should be used by third-party code etc.